src/Tribe/RSVP.php

```diff
@@ -61,6 +61,13 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	public $security_code = '_tribe_rsvp_security_code';
 
 	/**
+	 * Meta key that if this attendee wants to show on the attendee list
+	 *
+	 * @var string
+	 */
+	public $optout = '_tribe_rsvp_attendee_optout';
```

Comment:
It won't cause any Back compatibility for users, but on QA anyone with data, will loose it, might make the QA a little bit funky.
I have no reasoning to have it be `optout` together, besides that whenever I see Opt-In in variables I normally see it as a `optin`. We could change it